jobs/ml-ops/container/inference-api/main.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import FastAPI
+from dotenv import load_dotenv
+import data_processing as process
+import pickle, boto3, pandas, os
+
+app = FastAPI()
+
+load_dotenv(dotenv_path='./.env')
```

Comment:
beware of this, it's not safe to upload secrets to a docker image. use Serverless Containers secrets for this.